embeddings/pipeline/lightning_classification.py

```diff
@@ -58,17 +54,20 @@ def __init__(
         self.task_model_kwargs = task_model_kwargs = initialize_kwargs(
             self.DEFAULT_TASK_MODEL_KWARGS, task_model_kwargs
         )
-        self.early_stopping_kwargs = initialize_kwargs(
-            self.DEFAULT_EARLY_STOPPING_KWARGS, early_stopping_kwargs
-        )
         self.task_model_kwargs.update(
             {"train_batch_size": train_batch_size, "eval_batch_size": eval_batch_size}
         )
+        self.early_stopping_kwargs = initialize_kwargs(
+            self.DEFAULT_EARLY_STOPPING_KWARGS, early_stopping_kwargs
+        )
+        self._logging_kwargs = initialize_kwargs(self.DEFAULT_LOGGING_KWARGS, logging_kwargs)
```

Comment:
whats the reason for defining only logging kwargs as a private attribute?

Reply:
I need to access it in the abstract class, so I created the abstract property `logging_kwargs` that returns self._logging_kwargs in children. It is not a perfect solution, but a better one probably will require a bigger refactor.

Reply:
moved to abstract class